packages/sw-build/src/lib/generate-sw.js

```diff
@@ -43,6 +43,9 @@ const errors = require('./errors');
  * @param {String} [input.cacheId] An optional ID to be prepended to caches
  * used by sw-build. This is primarily useful for local development where
  * multiple sites may be served from `http://localhost`.
+ * @param {Boolean} [input.clientsClaim] An optional boolean that indicates if
+ * the page should skip waiting and claim current pages on a new service worker
```

Comment:
Sorry - this was a typo before I realised you had split out clients claim and skipwaiting options.